### PR TITLE
Fix login issue on iOS 10 simulator in Xcode 10

### PR DESCRIPTION
**Pull request by RickMohr**
As suggested in the SO post referenced in #326 I turned on the "Keychain Sharing" switch in the Capabilities section of Build Settings.

Connects #326

### CODE REVIEW

File: OpenTreeMap/OpenTreeMap.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1048,7 +1053,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run \'pod install\' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
```
Comment (jwalgran) [inline OpenTreeMap/OpenTreeMap.xcodeproj/project.pbxproj]: This change to `\'` escaping CocoaPods shell script does not look related to the keychain change. It is ok to mix it in, but it would be good to explain why the change was required.

Comment (RickMohr) [inline OpenTreeMap/OpenTreeMap.xcodeproj/project.pbxproj]: Those were presumably inserted by Xcode when saving the project file. Added a comment as such.
